main.sh

```diff
@@ -46,6 +46,11 @@ else
   . "$dir/../@google-cloud/cloud-rad/generate-devsite.sh"
 fi
 
+if [[ -n "$NO_UPLOAD" ]]; then
```

Comment:
Should this be set in the test suite? wondering why you added it.

Reply:
Yes, added this to our test command. This is so that we don't attempt to upload docs when running unit tests.